### PR TITLE
spinner: Ensure there are no logs/prints between Start and Stop

### PR DESCRIPTION
**Pull request by blanquicet**
I just learned that Stop cleaned up the spinner and any prefix/suffix message you add to it, so I think we can use it while still keeping the final output clean:

#### Example 1:

```bash
$ ./kubectl-aks config import
/ Importing...WARN[0030] Could not get VMSS VMs via Kubernetes API
WARN[0030] Please provide '--subscription', '--resource-group' and '--cluster-name' flags to get VMSS VMs via Azure API
Error: getting VMSS VMs via Kuberntes API: listing nodes: Get "https://172.28.128.4:6443/api/v1/nodes": dial tcp 172.28.128.4:6443: i/o timeout
```

#### Example 2:

```bash
$ ./kubectl-aks check-apiserver-connectivity
/ Checking connectivity...Connectivity check: failed with returned value 1
```

#### Example 1:

```bash
$ ./kubectl-aks config import
WARN[0030] Could not get VMSS VMs via Kubernetes API
WARN[0030] Please provide '--subscription', '--resource-group' and '--cluster-name' flags to get VMSS VMs via Azure API
Error: getting VMSS VMs via Kuberntes API: listing nodes: Get "https://172.28.128.4:6443/api/v1/nodes": dial tcp 172.28.128.4:6443: i/o timeout
```

#### Example 2:

```bash
$ ./kubectl-aks check-apiserver-connectivity
Connectivity check: failed with returned value 1:
```

### CODE REVIEW

File: cmd/check-apiserver-connectivity.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ import (
 var connCheckCmd = &cobra.Command{
 	Use:          "check-apiserver-connectivity",
 	Short:        "Check connectivity between the nodes and the Kubernetes API Server",
+	Args:         cobra.NoArgs,
```
Comment (mqasimsarfraz) [inline cmd/check-apiserver-connectivity.go:18]: nit: Do we need this for `version` command as well?

File: cmd/config.go [hunk 2]
```diff
@@ -125,20 +125,27 @@ func importCmdCommand() *cobra.Command {
 	var clusterName string
 
 	virtualMachineScaleSetVMs := func() (map[string]*utils.VirtualMachineScaleSetVM, error) {
+		utils.DefaultSpinner.Start()
+		utils.DefaultSpinner.Suffix = " Importing..."
+
 		if subscriptionID != "" && resourceGroup != "" && clusterName != "" {
```
Comment (mqasimsarfraz) [inline cmd/config.go:131]: nit: I wonder if it still makes sense to do `defer utils.DefaultSpinner.Stop()` here and continue to stop the spinner where explicitly required?

**Review by mqasimsarfraz — APPROVED** [review]
LGTM!

File: cmd/config.go (hunk 2 above)
Comment (blanquicet) [reply cmd/config.go:131]: If we defer it and there is an error in `VirtualMachineScaleSetVMsViaKubeconfig`. Then, we will fall in the example 1:

```bash
$ ./kubectl-aks config import
/ Importing...WARN[0030] Could not get VMSS VMs via Kubernetes API
WARN[0030] Please provide '--subscription', '--resource-group' and '--cluster-name' flags to get VMSS VMs via Azure API
Error: getting VMSS VMs via Kuberntes API: listing nodes: Get "https://172.28.128.4:6443/api/v1/nodes": dial tcp 172.28.128.4:6443: i/o timeout
```

File: cmd/check-apiserver-connectivity.go (hunk 1 above)
Comment (blanquicet) [reply cmd/check-apiserver-connectivity.go:18]: Done

File: cmd/config.go (hunk 2 above)
Comment (mqasimsarfraz) [reply cmd/config.go:131]: Ah, right! Got it!

**Review by mqasimsarfraz — APPROVED** [review]
LGTM! Since to learn about `cobra.NoArgs` :)

**Pull request merged**